Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -6,6 +6,13 @@ import BraintreeCore
 class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
 
     lazy var payPalClient = BTPayPalClient(apiClient: apiClient)
+    
+    lazy var emailTextField: UITextField = {
+        let textField = UITextField()
+        textField.placeholder = "placeholder@email.com"
+        textField.backgroundColor = .systemBackground
+        return textField
+    }()
 
```

Comment:
Does this work now for any valid sandbox accounts or are there test accounts to use?
Are there same test accounts for app switch and for vault?

Reply:
Great question! Currently the App Switch flow only works for a _specific_ test account, which is why you see a hardcoded tokenization key. Soon, it will work for all sandbox accounts.